From 61d1719485eaa74cad5ba9d73e0a25fa288f54f0 Mon Sep 17 00:00:00 2001
From: Ivan Sein <ivan@struktur.de>
Date: Wed, 25 Mar 2015 17:00:59 +0100
Subject: [PATCH 35/56] plugin: only load plugins from user's domain on Mac OS

This patch prevents Iridium from loading plugins that are already
installed on your machine (/Library/Internet Plug-Ins).
So Iridium will only load plugins from user's domain (~/Library/Internet
Plug-Ins).
---
 content/common/plugin_list_mac.mm | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/content/common/plugin_list_mac.mm b/content/common/plugin_list_mac.mm
index b3d726c..6fa826c 100644
--- a/content/common/plugin_list_mac.mm
+++ b/content/common/plugin_list_mac.mm
@@ -282,9 +282,6 @@ void PluginList::GetPluginDirectories(
 
   // Load from the user's area
   GetPluginCommonDirectory(plugin_dirs, true);
-
-  // Load from the machine-wide area
-  GetPluginCommonDirectory(plugin_dirs, false);
 }
 
 void PluginList::GetPluginsInDir(
-- 
2.4.3

